frontend/src/MuiForms/CustomTextInput.tsx

```diff
@@ -12,40 +14,48 @@ interface EntitySelectorProps {
   InputProps?: any
 }
 
-export default function EntitySelector(props: EntitySelectorProps) {
+export default function CustomTextInput(props: CustomTextInputProps) {
   const { onChange, value: currentValue, label, formContext } = props
 
-  const _onChange = (e) => {
-    onChange(e.target.value)
-  }
-
   const theme = useTheme()
 
+  const _onChange = (event: React.ChangeEvent<HTMLInputElement>) => {
```

Comment:
We use `_` to denote something as being intentionally unused when we have no choice but to provide the value (e.g. when we only care about the second parameter in a function). Consider removing the `_`